docs/workflows/api-token.md

```diff
@@ -546,3 +546,48 @@ In order to validate a token offline, the JWT header and claims must be decoded.
 4. Check timestamps in JWT claims where `nbf` <= current time as unix timestamp in seconds <= `exp`
 5. Check that JWT claims target tenant `bi_t` and target realm `bi_r` match the tenant and realm for the given application.
 6. Check that JWT claims has the expected scopes.
+
+## Listing access tokens
+
+Our API also provides the functionality to list active tokens that are held by a given identity or an application.
+In order to list tokens, you need to know the ID of the application that was used to issue the token.
+
+You need to use a Management API Bearer token with the `tokens:read` scope in order to call this API.
+
+### Request: Listing access tokens for an identity
+
+```bash
+curl 'https://api-${REGION}.beyondidentity.run/v1/tenants/${TENANT_ID}/realms/${REALM_ID}/tokens?application=${APPLICATION_ID}&principal_type=identity&principal_id=${IDENTITY_ID}'
+-H "Authorization Bearer ${MANAGEMENT_API_TOKEN}"
+```
+
+### Request: Listing access tokens for an application
+
+```bash
+curl 'https://api-${REGION}.beyondidentity.run/v1/tenants/${TENANT_ID}/realms/${REALM_ID}/tokens?application=${APPLICATION_ID}&principal_type=application&principal_id=${APPLICATION_ID}'
+-H "Authorization Bearer ${MANAGEMENT_API_TOKEN}"
+```
+
+In this case, the application is both the issuer and the subject of the token,
+which is why it needs to be in the request twice.
```

Comment:
I would move this up to under the header & above the bash example 
I would also add a similar comment for the "Listing access tokens for an identity" as the `principal_id`

```suggestion
When listing access tokens for an application, the application need to be in the request twice as it is both the issuer `application` and the subject `principal_id` of the token. 
```